<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BainComplexInstallerDialog</class>
 <widget class="QDialog" name="BainComplexInstallerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BAIN Package Installer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>This looks like a Package prepared for Installation through BAIN. You can select options from the list below. If there is a package.txt file, it should contain detailed information about the options.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="optionsList">
     <property name="toolTip">
      <string>Components of this package.</string>
     </property>
     <property name="whatsThis">
      <string>Components of this package.
If there is a component called &quot;00 Core&quot; it is usually required. Options are ordered by priority as set up by the author.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="packageBtn">
       <property name="toolTip">
        <string>The package.txt is often part of BAIN packages and contains details about the options available.</string>
       </property>
       <property name="whatsThis">
        <string>The package.txt is often part of BAIN packages and contains details about the options available.</string>
       </property>
       <property name="text">
        <string>Package.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="manualBtn">
       <property name="toolTip">
        <string>Opens a Dialog that allows custom modifications.</string>
       </property>
       <property name="whatsThis">
        <string>Opens a Dialog that allows custom modifications.</string>
       </property>
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
